### PR TITLE
CRYPTO-53: Fix assembly, pom.xml, RELEASE-NOTES, README, NOTICE.txt...etc

### PR DESCRIPTION
**Pull request by sundapeng**
Fixing https://issues.apache.org/jira/browse/CRYPTO-53
Fix assembly, pom.xml, RELEASE-NOTES, README, NOTICE.txt...etc for release

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+              Apache Commons Crypto 1.0.0-SNAPSHOT
```
Comment (winningsix) [inline RELEASE-NOTES.txt]: We should generate this file in release branch.

**Comment by sundapeng** [comment]
Thank @winningsix , patch update

**Comment by winningsix** [comment]
LGTM Thanks @sundapeng

**Comment by sundapeng** [comment]
Thank @winningsix

**Pull request closed without merging**